public/components/common/hooks/index.ts

```diff
@@ -25,5 +25,6 @@ export * from './use-app-config';
 export * from './useRootScope';
 export * from './use_async_action';
 export * from './use_async_action_run_on_start';
+// export * from './use-previous';
```

Comment:
**nitpick**: remove this comment

Reply:
Deleted in commit [df1295c](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/df1295c75871f851905f986d672b25fdf12af007)